corehq/apps/domain/forms.py

```diff
@@ -420,6 +421,12 @@ class DomainGlobalSettingsForm(forms.Form):
         help_text=gettext_lazy("Name of the project to be used in SMS sent for account confirmation to users.")
     )
 
+    release_mode_visibility = BooleanField(
+        label=gettext_lazy("Enable Release Mode"),
+        required=False,
+        help_text=gettext_lazy("Check this box to enable release mode in the app settings.")
```

Comment:
A link to docs or a longer explanation would be helpful.

Reply:
I think we should add a confluence page for this since its a public facing feature and link that here.
@orangejenny I am guessing this would be the recommended usual path anyway for user facing features?

Reply:
yeah, that's a good point - the new description is helpful, **and** linking to docs is also always a good idea